internal/provider/inventory_resource_test.go

```diff
@@ -215,6 +212,17 @@ resource "aap_inventory" "test" {
 }`, name)
 }
 
+// testAccInventoryResourceWithOrg returns a configuration for an AAP Inventory with the provided name, organization and all options.
+func testAccInventoryResourceWithOrg(name string, org string) string {
+	return fmt.Sprintf(`
+resource "aap_inventory" "test" {
+  name = "%s"
+  description = "A test inventory"
+	organization = "%s"
```

Comment:
```suggestion
  organization = "%s"
```
The indentation (tab) made it harder to see this